python/lsst/ts/tcpip/one_client_server.py

```diff
@@ -172,6 +156,16 @@ def connected(self) -> bool:
             or self.writer.is_closing()
         )
 
+    async def _monitor_connection(self):
+        """Monitor to detect if the client drops the connection.
+
+        For this to work, code must try to read data from the reader.
```

Comment:
Which `code` do you mean here? Certainly not the `_monitor_connection` function since it doesn't try to read data from the reader. What happens if the `code` doesn't try to read data?

Reply:
Ah, I need to delete this line. Thanks!